infrastructure/template.yaml

```diff
@@ -163,21 +163,37 @@ Resources:
       BuildProperties:
         Sourcemap: true
     Properties:
-      Handler: lambdas/bearer-token-handler/src/bearer-token-handler.ts
+      Handler: lambdas/bearer-token-handler/src/bearer-token-handler.lambdaHandler
+      LoggingConfig:
+        LogGroup: !Sub /aws/lambda/${AWS::StackName}/BearerTokenHandlerFunction
       CodeSigningConfigArn:
         !If [EnforceCodeSigning, !Ref CodeSigningConfigArn, !Ref AWS::NoValue]
 
+  BearerTokenHandlerFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub /aws/lambda/${AWS::StackName}/BearerTokenHandlerFunction
+      RetentionInDays: !If [IsDevLikeEnvironment, 7, 30]
```

Comment:
Please can you switch this, like Tim did, to [IF PROD 30, 7]

Reply:
I copied Tim's code from here: https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/5d1924099fc4c604118b47d1ada7ed0214fda436/infrastructure/template.yaml#L206 

Would be better to stay consistent with the other code

Reply:
Looks like we are not consistent anyway https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/5d1924099fc4c604118b47d1ada7ed0214fda436/infrastructure/template.yaml#L607
I thought he changed it everywhere, when I saw this I thought this is better do we really need more than 7 days in all the other environments?